code/modules/mob/mob.dm

```diff
@@ -1442,4 +1442,14 @@ mob/proc/yank_out_object()
 		M.pixel_y = initial(M.pixel_y)
 
 /mob/proc/can_unbuckle(mob/user)
-	return 1
\ No newline at end of file
+	return 1
+
+
+//Can the mob see reagents inside of containers?
+/mob/proc/can_see_reagents()
+	if(ishuman(src))
```

Comment:
Because on TG, it works for more than just humans; I deliberately cut those, but intentionally left it at the mob level in case we want to make changes to it, in the future.